### DISCUSSION TITLE
Novel rating scale plugin

### DISCUSSION THREAD

**Discussion by jackransomlovell** [jspsych/jsPsych#709] [Q&A]
Hi all, I am an undergrad RA in the Wager Lab at Boulder. We developed a novel rating scale (see below) for an fMRI study investigating functional alignment. We have a Matlab function that works very well and was developed in Matlab's psychtoolbox, but we are hoping to implement it online somehow (for MTurk experiments). Ideally, it could become a new plugin for the jspsych library! If anyone is interested in helping me with this I would greatly appreciate it, or just any advice, in general, would be very useful :). 
![image](https://user-images.githubusercontent.com/50260796/82358950-3ab14000-99d5-11ea-8dc9-411ab0ba7eda.png)

**Comment by jodeleeuw** [comment]
This looks neat! How do users interact with the rating scale? Clicking somewhere?

**Reply by jackransomlovell** [reply]
Oops! I'm sorry I must have not seen this!! 
Yes, the participant will click on the rating scale to give their rating. Their cursor appears as a little grey dot. Their cursor is disabled upon leaving the scale. 
I am planning on making a scope of work for this project sometime this week. I will post it here if anyone would like to help out! :)

**Reply by jkhartshorne** [reply]
@jack-rl-ovell any updates?

**Comment by jackransomlovell** [comment]
No, I've been busy developing another online task for the lab. This would
be my first time developing something using html/css/java. I need to sit
down and hash out a plan. I imagine my first step would be writing the css
and js files for the presentation and function of the rating scale.
